BlogTemplate/wwwroot/js/site.js

```diff
@@ -1 +1,18 @@
-﻿// Write your Javascript code.
+﻿(function () {
```

Comment:
I'm not sure if this should go into site.js.  This function is only relevant to a couple pages, and never relevant to someone reading the site content, so there's not value in serving it up in that case.